### ISSUE TITLE
Add option for providing script to check that any version file was updated

### ISSUE THREAD

**Issue by christophebedard** [christophebedard/tag-version-commit#95]
Generally, when a new release is created, some kind of version file is modified to update the version number, e.g. `__version__ = 1.2.3` in a Python module's `__init__.py` file.

An option for users to provide their own script to check that the to-be-created tag matches their application's version would be useful. For example, for a Python module called `my_module`:

```
$ python3 -c "import my_module; assert my_module.__version__ == '1.2.3'"
```

If that check fails, the action should fail. The version string to compare against should be the match for the `version_regex`, before prepending the `version_tag_prefix`.

The option could have a default value of `""` which would simply skip the check.

**Comment by christophebedard** [comment]
Closed via #111

**Comment by christophebedard** [comment]
Example: https://github.com/christophebedard/tag-version-commit/commit/d902eff84c4b99ff6c7d8b420f91c9e8bfa29f83

The commit is titled "1.0.4" but the commit didn't update the version in the `package.json` file, which is what the `version_assertion_command` checks in both tag/tag and test/test-action jobs, so they fail:

* https://github.com/christophebedard/tag-version-commit/runs/1045652656
* https://github.com/christophebedard/tag-version-commit/runs/1045652680

I then updated the `package.json` file so that the version matches and all actions/jobs pass: https://github.com/christophebedard/tag-version-commit/commit/0a1cbb14707ecf36d16aeb2241d4d8cf80e8f773
